prooph/event-store-bus-bridge/3.1/config/packages/prooph_event_store_bus_bridge.yaml

```diff
@@ -0,0 +1,19 @@
+services:
+    _defaults:
+        public: false
+
+    Prooph\EventStoreBusBridge\EventPublisher:
+        arguments:
+            - '@prooph_service_bus.default_event_bus'
+        tags:
+            - { name: 'prooph_event_store.default.plugin' }
+
+    # The app.event_store.default service must be configured with a
+    # Prooph\EventStore\TransactionalEventStore instance in order to user the following plugin.
+    # Read more: http://docs.getprooph.org/event-store/implementations/pdo_event_store/variants.html#3-2-1-2
+    #
+    # Prooph\EventStoreBusBridge\TransactionManager:
```

Comment:
I do not like config in comments like this. If you think that 90% of the users will use this config you should un-comment this. If not I think these lines should be removed.